### PR TITLE
tidying grammar

### PR DESCRIPTION
**Pull request by chauey**
minor grammar and formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,41 +16,41 @@
...
+**IMPORTANT**: All of the written above is a perfect vision of the end result; don't take it seriously from the start :)
 
-**NOTE**: 90% of requirments will be solved by unified compilation of source code, docs, gh-pages
+**NOTE**: 90% of requirements will be solved by unified compilation of the source code, docs, gh-pages
 
-1. Consistency with Angular2
-  1. semver consistency (1 to 1)
-  2. delivy consistency (es5 commonjs as npm, ts near to js, systemjs bundles)
+1. Consistency with Angular 2
+  1. server consistency (1 to 1)
```
Comment (valorkin) [inline README.md]: no, really, semver, I mean it :) http://semver.org/

**Comment by valorkin** [comment]
@chauey thank you so much for you effort, please revert `server` to `semver` and `git commit -a --amend` :)

**Comment by chauey** [comment]
Hi Dmitriy, you guys doing awesome work. 

 

It took time to figure out the amend. Hope it’s correct now.

**Comment by valorkin** [comment]
Looks good, thanks a lot :)
I will break again all the things with my grammar again soon
but I will try to not do this now :)

**Pull request merged**